locale/messages.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
```

Comment:
Could you revert all the translations (checkout the files from latest master commit), as this PR doesn't add any new translation strings, so this doesn't need to contain any translation changes?